### PR TITLE
Side nav ds pr

### PR DESCRIPTION
**Pull request by ray760**
#### What's this PR do?
1. Makes side navigation updates.
2. Moves side navigation css into navigation.scss file.
3. Makes updates to the core ds-css file.
4. Swaps out sass variables with CSS variables accessible from ds-css. 

#### Where should the reviewer start?
Pull in branch,. npm i and run locally to see side nav updates, and view navigation.scss to see updates.

#### How should this be manually tested?
View local site and scss files

#### Any background context you want to provide?
N/A

#### What are the relevant tickets?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
- Is there a blog post?
N/A
- Does the knowledge base need an update?
N/A

### CODE REVIEW

**Comment by minjungb** [comment]
Viewed the changes, and it seems that the highlighting part got moved a bit, as well as the color of the vertical line (PR has color #4a4a4a, production has #aaa98)
Ref below:
image

current highlighting:
image

File: ds-ux-guidelines/src/ds-components/navigation/navigation.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
+@mixin menu-list-ul-overide {
```
Comment (davezen1) [inline ds-ux-guidelines/src/ds-components/navigation/navigation.scss:1]: would it make sense to make a comment next to the !important as to the why we are doing it?

File: ds-ux-guidelines/src/ds-components/navigation/navigation.js [hunk 2]
```diff
@@ -113,28 +113,24 @@ class Navigation extends Component {
...
                 <li>
-                  <Link to={subCat.link}>
-                    <span className="menu-text">{subCat.name}</span>
-                  </Link>
+                  <Link to={subCat.link}><span className="menu-text cbp-body">{subCat.name}</span></Link>
 
                   {subCat.anchors !== undefined ||
                   subCat.anchors !== undefined ? (
-                    <ul style={{ borderLeft: "solid 1px #aaaa98" }}>
+                    <ul style={{ borderLeft: "solid 1px #4a4a4a" }}>
```
Comment (davezen1) [inline ds-ux-guidelines/src/ds-components/navigation/navigation.js:125]: can we pull this out into a variable?

File: ds-css/src/styles/custom/utilities/initial-variables.scss [hunk 3]
```diff
@@ -70,7 +70,7 @@ $cbp-warning-text:                 #a44900;
 $link-hover:                      #001b44;
 // Typography
 /* CBP Default Font Family */
-$family-sans-serif: "Roboto", "Calibri", "Tahoma", sans-serif;
+$family-sans-serif: "Roboto", "Calibri", "Tahoma", sans-serif !important;
```
Comment (davezen1) [inline ds-css/src/styles/custom/utilities/initial-variables.scss:73]: Why is this necessary? (!important)

File: ds-ux-guidelines/src/ds-components/navigation/navigation.scss [hunk 4]
```diff
@@ -36,6 +38,10 @@ a.dropdown-item {
   font-size: var(--cbp-body-text);
   padding: 0;
   margin: 0;
+
+  span {
+    padding-left: var(--cbp-sm-space);
```
Comment (minjungb) [inline ds-ux-guidelines/src/ds-components/navigation/navigation.scss:45]:
```suggestion
    padding-left: var(--cbp-reg-space);
```

File: ds-ux-guidelines/src/ds-components/navigation/navigation.scss [hunk 5]
```diff
@@ -8,12 +9,13 @@
...
 .nav-wrapper {
   background-color: var(--grey-lightest);
-  padding-left: 2rem;
+  // padding-left: 2rem;
 }
 
 .dropdown-container {
   margin-bottom: 2rem;
   margin-top: 8px;
+  padding-left: var(--cbp-sm-space);
```
Comment (minjungb) [inline ds-ux-guidelines/src/ds-components/navigation/navigation.scss:20]:
```suggestion
  padding-left: var(--cbp-reg-space);
```

File: ds-ux-guidelines/src/ds-components/navigation/navigation.scss [hunk 6]
```diff
@@ -1,5 +1,6 @@
 @mixin menu-list-ul-overide {
-  margin-left: 0 !important;
+  margin-left: var(--cbp-sm-space) !important;
```
Comment (minjungb) [inline ds-ux-guidelines/src/ds-components/navigation/navigation.scss:4]:
```suggestion
  margin-left: var(--cbp-reg-space) !important;
```
